app/controllers/application_controller.rb

```diff
@@ -1,2 +1,23 @@
 class ApplicationController < ActionController::Base
+  include Pagy::Backend
+
+  protect_from_forgery with: :exception
+
+  before_action :authenticate_user!, except: %i[index show]
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
```

Comment:
- [ ] We are supposed to set the `current_user` here in this method:

```suggestion
def current_user
  User.first
end

```